streetscape-version-1-overview.md

```diff
@@ -0,0 +1,82 @@
+# Streetscape 1
+
+Building the foundation for our design system to sync designers and developers workflows to create consistent and accessible digital products.
+
+> [!IMPORTANT]
+> Although the sections below are broken into designers and developers portions for easy reference, it is encouraged for everyone to take the time to understand and learn the different parts that contribute to Streetscape.
+
+## For Designers
+
+### Figma: Libraries and Design Tokens
+
+Within Figma, we have established six foundational libraries that contain design styles respective to their category. Each style contain a design token, which are reusable values used to sync design and development. The importance of design tokens is it establishes collective agreement on design decisions, speeding up the design and development process.
+
+All Figma files related to Streetscape live in the Streetscape project.
+
+![alt-text](https://github.com/NYCPlanning/design/blob/streetscape-v1/assets/streetscape-v1/streescape%20figma%20project.png)
+
+To learn more about how to access and update libraries when working on a design file as well as overall how Figma is used, refer to [Streetscape: Getting Started](https://www.figma.com/design/mfEZjCGYOX49VKyptVDLG6/Streetscape%3A-Getting-Started?node-id=0-1&t=cP49FaXFBwrFe1i0-1).
+
+> [!NOTE]
+> Each library file within the Streetscape project contains documentation about design decisions and best practices.
+
+### EDDE Redesign: Test and Iterate
+
+The best method to make sure we on the right track with our decision making while developing our libraries was to apply them to a design. Therefore, we applied our libraries to an EDDE redesign. Refer to [EDDE Redesign](https://www.figma.com/design/d9FEjwAyl6Mb4Sep9tPdNt/EDDE-Redesign?node-id=6-108&t=ZUED3d3k3Qltot4j-1) to look at comparison between the previous and current design.
+
+## For Developers
+
+### Storybook: Upgrading to Version 7.6.2
+
+Updating to the latest 7 version of Storybook to implement Storybook Designs Plug-In, a feature that allows us to embed Figma files into a Storybook component.
```

Comment:
could probably drop this section since the plug-in guide below mentions the necessary version and it's fair to assume a dev using the streetscape repo is using whatever's on main